openedx/core/lib/api/view_utils.py

```diff
@@ -118,3 +121,57 @@ def _decorator(func_or_class):
             func_or_class.permission_classes += (IsUserInUrl,)
         return func_or_class
     return _decorator
+
+
+def add_serializer_errors(serializer, data, field_errors):
```

Comment:
Was this method taken from user_api? If so, can you remove it from there and call the new utility method?